includes/Meta_Boxes.php

```diff
@@ -0,0 +1,185 @@
+<?php
+/**
+ * Class Meta_Boxes.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+use WP_Screen;
+
+/**
+ * Class Meta_Boxes.
+ */
+class Meta_Boxes {
+	/**
+	 * Supported meta box locations.
+	 *
+	 * @var string[]
+	 */
+	const LOCATIONS = [ 'normal', 'advanced' ];
+
+	/**
+	 * Meta box priorities.
+	 *
+	 * @var string[]
+	 */
+	const PRIORITIES = [ 'high', 'sorted', 'core', 'default', 'low' ];
+
+	/**
+	 * Init.
+	 *
+	 * @since 1.2.0
+	 *
+	 * @return void
+	 */
+	public function init() {
+		add_action( 'add_meta_boxes_' . Story_Post_Type::POST_TYPE_SLUG, [ $this, 'remove_meta_boxes' ], PHP_INT_MAX );
```

Comment:
This feel odd. Can we make it is class property like this. 

```
add_action( 'add_meta_boxes_' . $this->post_type, [ $this, 'remove_meta_boxes' ], PHP_INT_MAX );
```

```
$this->meta_boxes = new Meta_Boxes( Story_Post_Type::POST_TYPE_SLUG );
```


Meaning in the future / in tests we could add another CPT.

Reply:
Or better yet, this line does live here at all. 

In  Story_Post_Type::init 

```
add_action( 'add_meta_boxes_' . self::POST_TYPE_SLUG, [ $this->meta_box, 'remove_meta_boxes' ], PHP_INT_MAX );
```

Reply:
Found it unnecessary, but sure we can do that.

Reply:
> Or better yet, this line does live here at all.
> 
> In Story_Post_Type::init
> 
> ```
> add_action( 'add_meta_boxes_' . self::POST_TYPE_SLUG, [ $this->meta_box, 'remove_meta_boxes' ], PHP_INT_MAX );
> ```

I see your point, but it's quite different from all our other classes, no? All other classes register their hooks in their `init()` method.

(Aside: If we can't agree on the best way right away, happy to open a dedicated ticket to address this. I really wanna merge it soon :)

Reply:
We could convert this into a trait and just use this 
```
add_action( 'add_meta_boxes_' . self::POST_TYPE_SLUG, [ $this, 'remove_meta_boxes' ], PHP_INT_MAX );
```

Reply:
I don't think a trait is a good solution for this kinda scenario